### DISCUSSION TITLE
How is SSL Custom Error Page Working with Cloudflare For Teams IP

### DISCUSSION THREAD

**Discussion by nilava** [AdguardTeam/AdGuardHome#4224] [Q&A]
If we set blocking mode to Custom IP and set them to the following

IPv4 - 162.159.36.12
IPv6 - 2606:4700:54::a29f:240c

I found them via nslookup for domains blocked via cloudflare for teams policy routing.

They work and redirect SSL websites to a custom page with no SSL certificate errors! 

How could this be done or replicated locally?

**Comment by ainar-g** [comment]
Hello.  I cannot reproduce that from outside of Cloudflare For Teams, at least in Firefox.  I get `SEC_ERROR_UNKNOWN_ISSUER`.  So, perhaps, they simply add their own CA to essentially perform a MITM when a policy is triggered?

**Reply by ainar-g** [reply]
No, I am unable to reproduce that.
